src/ucp/core/ucp_ep.c

```diff
@@ -231,6 +231,8 @@ ucs_status_t ucp_ep_create(ucp_worker_h worker,
     uint64_t dest_uuid;
     ucp_ep_h ep;
 
+    UCS_STATIC_ASSERT(sizeof(*ep_p) < 160); /* 10 cache lines */
```

Comment:
@hoopoepg i think the intention was to do it in gtest, and use '==' comparison
also 160 is 3 cache lises

Reply:
yes, gtest will check this values too, but it will require including of internal headers, and I'm not sure if it is possible check for sizes for UCT layer (we will have to include a lot of internal headers from UCT)